test/template_t.cpp

```diff
@@ -67,19 +67,10 @@ using namespace OpenOrienteering;
 namespace
 {
 
-QPointF center(const TemplateTrack* temp)
-{
-	QPicture picture;
-	QPainter painter(&picture);
-	temp->drawTracks(&painter, false);
-	painter.end();
-	return QRectF(picture.boundingRect()).center();
-}
-
 QPointF center(const Template* temp)
 {
 	if (auto* template_track = qobject_cast<const TemplateTrack*>(temp))
-		return center(template_track);
+		return template_track->getCenter();
```

Comment:
I found that I can simply use `template_track->calculateTemplateBoundingBox().center()`. It differs from the proposed `TemplateTrack::getCenter()` by:
- handling also waypoints.
- calling `templateToMap` for each point, instead of for the final result.

Given the low number of points in the test data, I will adopt to use the existing function, and not add `getCenter()`.